tmd/Topology/methods.py

```diff
@@ -145,10 +145,10 @@ def get_ph_angles(tree, feature='radial_distances', **kwargs):
     angles = get_angles(tree, beg, parents, children)
 
     while len(np.where(active)[0]) > 1:
-        alive = list(np.where(active)[0])
-        for l in alive:
+        alives = list(np.where(active)[0])
```

Comment:
why does `alives` need to be converted from an np.array to a list?

Reply:
I have the impression this conversion to list is no longer necessary.